mu4/domain/notation/internal/notation.cpp

```diff
@@ -74,6 +65,14 @@ Notation::Notation(Score* score)
 
 Notation::~Notation()
 {
+    delete m_interaction;
+    delete m_midiInput;
+    delete m_accessibility;
+    delete m_undoStackController;
+    delete m_style;
+    delete m_playback;
+    delete m_elements;
+
```

Comment:
Why all these pointers are not shared?

Reply:
I agree, need to make shared. Better do this as a separate task.